packages/ui-extensions/docs/surfaces/checkout/staticPages/upgrading-to-2025-10.doc.ts

```diff
@@ -35,29 +39,29 @@ These web components are an early access preview of the [Polaris](https://shopif
-|   \`ConsentPhoneField\`      |   \`ConsentPhoneField\`                                   |   Coming Soon                          |
-|   \`DateField\`              |   \`DateField\`                                           |   Coming Soon                          |
-|   \`DatePicker\`             |   \`DatePicker\`                                          |   Coming Soon                          |
-|   \`Disclosure\`             |   \`Details\` and \`Summary\`                         |   Coming Soon                          |
-|   \`Divider\`                |   \`Divider\`                                             |   Coming Soon                          |
+|   \`ConsentCheckbox\`        |   \`ConsentCheckbox\`                                     |   Coming soon                          |
+|   \`ConsentPhoneField\`      |   \`ConsentPhoneField\`                                   |   Coming soon                          |
+|   \`DateField\`              |   \`DateField\`                                           |   Coming soon                          |
+|   \`DatePicker\`             |   \`DatePicker\`                                          |   Coming soon                          |
+|   \`Disclosure\`             |   \`Details\` and \`Summary\`                         |   Coming soon                          |
+|   \`Divider\`                |   \`Divider\`                                             |   Coming soon                          |
 |   \`DropZone\`               |   [DropZone](/api/checkout-ui-extensions/2025-10-rc/components/forms/dropzone)                                            |   Available today                      |
 |   \`Form\`                   |   [Form](/api/checkout-ui-extensions/2025-10-rc/components/forms/form)                                                |   Available today                      |
-|   \`Grid\`                   |   \`Grid\`                                                |   Coming Soon                          |
-|   \`GridItem\`               |   \`GridItem\`                                            |   Coming Soon                          |
+|   \`Grid\`                   |   \`Grid\`                                                |   Coming soon                          |
+|   \`GridItem\`               |   \`GridItem\`                                            |   Coming soon                          |
 |   \`Heading\`                |   [Heading](/api/checkout-ui-extensions/2025-10-rc/components/titles-and-text/heading)                                             |   Available today                      |
-|   \`HeadingGroup\`           |   \`Section\`                                             |   Coming Soon                          |
+|   \`HeadingGroup\`           |   \`Section\`                                             |   Coming soon                          |
```

Comment:
We are exposing Section in the end, for Customer Account.

Reply:
true, we should let osaru know, and will update here